### ISSUE TITLE
timelines endpoint

### ISSUE THREAD

**Issue by k0a1a** [bear/python-twitter#237]
are you plannoing to support https://api.twitter.com/1.1/timelines/timeline.json endpoint?

**Comment by bear** [comment]
which API call is that? I don't see any mention of /timelines in the current API docs - can you give me the API docs link?

**Comment by k0a1a** [comment]
my bad, 'timelines/timeline.json' is an obsolete name for https://api.twitter.com/1.1/collections/entries.json which is described here https://dev.twitter.com/rest/reference/get/collections/entries

**Comment by ryankicks** [comment]
I did a fork and added some of this here: https://github.com/ryankicks/python-twitter

If @k0a1a is interested in merging or helping me merge?

**Comment by k0a1a** [comment]
@ryankicks could you point me to collections relevant code in your fork?

**Comment by ryankicks** [comment]
Absolutely. My fork was just in preparation (a year ago) so the latest entries are collections related. I just updated the latest code to reflect the new endpoints and the use of Collections vs. Custom Timelines (the old terminology)

https://github.com/ryankicks/python-twitter/commits/master

**Comment by bear** [comment]
say the word and I'll add you to the collaborator's list so you both can work on this new code

**Comment by ryankicks** [comment]
I am not confident in my Python SKILLZ enough to be worthy to submit. O_O

**Comment by bear** [comment]
submit it via a PR and we can review it

**Comment by k0a1a** [comment]
@ryankicks thanks for pushing the updates, appreciated. I'm going to see if I can contribute.
@bear neither too confident to add code right away but will keep you informed on my progress

**Comment by ryankicks** [comment]
I should also add that the Collections endpoints require additional privileges to access. Feel free to mail me (rchoi AT twitter.com) in advance the API tokens so I can whitelist them for your/any testing.

I'll also add this to the code itself so it's more clear how to use them. Sound good?

**Comment by bear** [comment]
thanks @ryankicks !  Please do file an issue against me for any code or doc updates you need.
